frontend/components/feedback-task/field/TextField.component.vue

```diff
@@ -38,6 +38,19 @@ export default {
       default: false,
     },
   },
+  computed: {
+    searchValue() {
```

Comment:
From my point of view, the best practice for that is receiving that value as a prop.